internal/graphicsdriver/opengl/context_js.go

```diff
@@ -69,6 +69,10 @@ func (p program) equal(rhs program) bool {
 	return p.value.Equal(rhs.value) && p.id == rhs.id
 }
 
+func (p program) getID() int {
```

Comment:
We can use existing `getProgramID`

Reply:
Do you mean:
```go
func (p program) getID() int {
	return int(getProgramID(p))
}
```

Because depending on arch, `program` still has to implement the same method and this only exists in context_js
Or do you mean providing `type programID int` + `getProgramID` across all context_os.go files ?

Reply:
Oh nvm, it does exist everywhere, I'll change